data/Makefile.am.inc

```diff
@@ -16,21 +16,23 @@ systemduserunit_DATA = xdg-desktop-portal-gtk.service
 CLEANFILES += $(systemduserunit_DATA)
 
 %.desktop.in: data/%.desktop.in.in
-	$(AM_V_GEN) sed -e "s|@libexecdir[@]|$(libexecdir)|" $< > $@
+	$(AM_V_GEN) msgfmt --desktop -d po --template $< -o $@
 
 %.desktop: %.desktop.in
-	$(AM_V_GEN) msgfmt --desktop -d po --template $< -o $@
+	$(AM_V_GEN) $(SED) -e "s|\@libexecdir\@|$(libexecdir)|" $< > $@
 
 desktopdir = $(datadir)/applications
 desktop_in_in_files = data/xdg-desktop-portal-gtk.desktop.in.in
-dist_desktop_DATA = xdg-desktop-portal-gtk.desktop
+desktop_in_files = xdg-desktop-portal-gtk.desktop.in
+desktop_DATA = xdg-desktop-portal-gtk.desktop
 
-CLEANFILES += $(dist_desktop_DATA)
+CLEANFILES += $(desktop_DATA)
 
 EXTRA_DIST += \
 	$(dbus_service_in_files)                \
         $(systemduserunit_in_files)             \
         $(desktop_in_in_files)                  \
+        $(desktop_in_files)                     \
```

Comment:
I usually get confused around these bits too, just checking I was not missing anything again. I'll merge this and then maybe land a follow up patch, after having tested it, just in case.